src/components/shell/ShellReadLine/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement } from 'react';
+
+import {
+  defaultPrefixStyle,
+  inputStyle,
+  readLineMessageStyle,
+  shellLineContainerStyle,
+} from './style';
+
+type ShellReadLineProps = {
+  prefix: 'question' | 'config';
```

Comment:
요거 단순히 생각해봤을 때 변경 가능성이 매우 높은 것 같은 데 굳이 타입으로 지정을 해줄 필요가 있을까 싶네요.... 그냥 string으로 지정해주고 `SHELL_PREFIX_QUESTION`, `SHELL_PREFIX_CONFIG` 같은 상수를 정의해서 사용하는 게 낫지 않을까 싶은 데 요건 단순히 제 개인적인 의견이므로 넘어가도 좋습니당.

```suggestion
  prefix: string;
```